doc/governance.md

```diff
@@ -13,6 +13,8 @@ The current SC consists of these members:
 - [@tomberek](https://github.com/tomberek) (term ends 2025)
 - [@winterqt](https://github.com/winterqt) (term ends 2025)
 
+The entire SC can be reached via <steering@nixos.org> or by pinging the [@NixOS/steering](https://github.com/orgs/nixos/teams/steering) GitHub team.
```

Comment:
There's also the [Discourse group](https://discourse.nixos.org/g/steering_committee), which AFAIK can be directly PM'd. Worth adding?

Reply:
It appears to not be possible, autocompletion doesn't bring up the SC. I believe the Discourse admins (#44 !) need to configure something for that to work. Notably the moderators can be PMd, but it autocompletes as "moderators", not ["moderation-team" which would be a Discourse group](https://discourse.nixos.org/g/moderation-team), which leads me to believe it's something different than Discourse groups that makes PMs possible.

Reply:
Whether or not a group can be mentioned and by whom is a per group setting. Right now nobody can mention the steering committee.